<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteCircle</class>
 <widget class="QMainWindow" name="DeleteCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Circle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Circle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOkCircle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
